### DISCUSSION TITLE
If a form or field conditionally disabled produces an error

### DISCUSSION THREAD

**Discussion by sarper57** [Budibase/budibase#8155] [Help]
Error image attached- I am using latest version of Budibase 2.0.21 i am not sure if this is a bug 

![2022-10-06](https://user-images.githubusercontent.com/42462205/194267612-cc4a9724-5efb-448f-8989-b3065ca1ab89.png)

**Comment by andz-bb** [comment]
Hey @sarper57, it looks like something has sent you to the wrong URL within the Budibase builder. Could you let me know what you were doing before you got this screen? I can have a look on my end and see if I can replicate it. Thanks

**Reply by sarper57** [reply]
Hi Andz-bb
Sorry for late reply I didn't get a notification. So before I got this error I was selecting a form from the menu and instead of displaying the page it produced the error.
However  this is probably due to the **On screen load** I have included a condition to disable part of the form component based on a if there were records available?

thanks
Sarper

**Reply by andz-bb** [reply]
Hi @sarper57, thanks for getting back to me. If you send an app export to support@budibase.com I can have a closer look at what's going on and see if I can recreate it on my end.
